internal/impl/devbox.go

```diff
@@ -348,14 +349,18 @@ func (d *Devbox) Exec(cmds ...string) error {
 	return nix.Exec(nixDir, cmds, env)
 }
 
-func (d *Devbox) PrintShellEnv() error {
-	profileBinDir, err := d.profileBinDir()
+func (d *Devbox) AddPluginEnv() error {
+	pluginEnvs, err := plugin.Env(d.cfg.Packages, d.projectDir)
 	if err != nil {
-		return errors.WithStack(err)
+		return err
+	}
+	script := ""
+	for _, pluginEnv := range pluginEnvs {
+		script += fmt.Sprintf("export %s\n", pluginEnv)
 	}
-	// TODO: For now we just updated the PATH but this may need to evolve
-	// to essentially a parsed shellrc.tmpl
-	fmt.Fprintf(d.writer, "export PATH=\"%s:$PATH\"", profileBinDir)
```

Comment:
Is path no longer needed?

Reply:
Correct. I'm using direnv's [use nix](https://direnv.net/man/direnv-stdlib.1.html#codeuse-nix-code) feature to get the nix environment. So no need to modify the path.